app/admin/users.rb

```diff
@@ -71,14 +71,29 @@ def to_html
       :timezone,
       :updated_at,
       :url_slug,
+      :is_over_18,
+      :consented_to_background_check,
+      :age_range,
+      :education,
+      :household_income,
+      :occupation_type,
+      :occupation,
       :timeout,
       language_ids: [],
       languages: [ :id, :name ],
       program_ids: [],
       programs: [ :id, :name ],
       reviews: [ :id, :description, :review ],
       role_ids: [],
-      roles: [ :id, :name ]
+      roles: [ :id, :name ],
+      main_goal_ids: [],
+      main_goals: [ :id, :name ],
+      meeting_option_ids: [],
+      meeting_options: [ :id, :name ],
+      gender_identity_ids: [],
+      gender_identities: [ :id, :name ],
+      ethnicity_race_ids: [],
+      ethnicity_races: [ :id, :name ]
```

Comment:
Is there a way we can allow the admin user to add/remove columns in active admin? Seems like there are more columns in here than someone could reasonably ready.